src/Contains.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * @template T
+ * @param {T} element
+ * @returns {boolean}
+ */
+function Contains(element) {
+  return this.indexOf(element) === -1 ? false : true;
```

Comment:
Does this need to be a ternary?

Reply:
The more type checking the better it would be.

Reply:
The ternary is not needed. Please change to `this.indexOf(element) !== -1`